### PR TITLE
use storage vm on network suite

### PR DESCRIPTION
**Pull request by hbraha**
Currently, network suite uses an engine for storage, using the storage VM
added in the previous patch.

### CODE REVIEW

File: common/deploy-scripts/setup_storage.sh [hunk 1]
```diff
@@ -52,8 +52,9 @@ set_selinux_on_nfs() {
 }
 
 setup_iscsi() {
-    # this is ugly, assumes that dedicated storage VMs (ost-[suite]-storage) use their primary network as storage network, and VMs with co-located engine have a dedicated storage network on eth1 (like basic-suite-master).
-    if [[ $(hostname) == *"-storage"* ]]; then
+    # this is ugly, assumes that dedicated storage VMs (ost-[suite]-storage) use their primary network as storage network only on HE suite, and VMs with co-located engine have a dedicated storage network on eth1 (like basic-suite-master).
+    echo $SUITE
+    if [[ $(hostname) == *"-storage"* ]] && [[ $(hostname) == *"he-"*  ]]; then
```
Comment (michalskrivanek) [inline common/deploy-scripts/setup_storage.sh:57]: why is this needed?

Comment (hbraha) [reply common/deploy-scripts/setup_storage.sh:57]: the HE suite already uses storage vm which have only eth0.

Comment (tinez) [reply common/deploy-scripts/setup_storage.sh:57]: We're planning to unify the NICes between HE suite and the others once we get these patches in

**Review by tinez — APPROVED** [review]

**Review by erav — APPROVED** [review]
